Main/StackWalker/StackWalker.cpp

```diff
@@ -1439,14 +1440,34 @@ void StackWalker::OnSymInit(LPCSTR szSearchPath, DWORD symOptions, LPCSTR szUser
     buffer[STACKWALK_MAX_NAMELEN - 1] = 0;
     OnOutput(buffer);
   }
+#elif (_MSC_VER >= 1800) && (_WIN32_WINNT > _WIN32_WINNT_VISTA)
+  OSVERSIONINFOEXW osInfo;

```

Comment:
You are mixing two types here… why? Please consistently use "RTL_OSVERSIONINFOEXW". Also osInfo can be declared inside the else...